python/spacewalk/satellite_tools/repo_plugins/yum_src.py

```diff
@@ -1243,6 +1250,7 @@ def set_download_parameters(self, params, relative_path, target_file=None, check
         params["timeout"] = self.timeout
         params["minrate"] = self.minrate
         params['proxies'] = get_proxies(self.proxy_url, self.proxy_user, self.proxy_pass)
+        params['logspec'] = self.urlgrabber_logspec 
```

Comment:
It seems we are missing to initialize the `self.urlgrabber_logspec` in cases where `CFG.urlgrabber_logspec` is None, so this will break here.

I would add `self.urlgrabber_logspec = None` in `ContentSource.__init__` function.